swag/swag_client.py

```diff
@@ -369,3 +416,335 @@ async def execute_swagdmin_command(self, message):
                 "du serveur\n"
                 "```"
             )
+
+    async def execute_cagnotte_command(self, message):
+        def get_cagnotte_idx_from_command(splited_command):
+            cagnotte_idx = int(
+                [
+                    identifiant[1:]
+                    for identifiant in splited_command
+                    if identifiant.startswith("€") and identifiant[1:].isnumeric()
+                ][0]
+            )
+            return cagnotte_idx
+
+        message_command = message.content
+        splited_command = message_command.split()
+
+        if "créer $wag" in message_command:
+            cagnotte_name = " ".join(splited_command[3:])
+            if len(cagnotte_name) == 0:
+                await message.channel.send(
+                    "Merci de mentionné un nom pour ta €agnotte."
```

Comment:
mentionner

Reply:
Resolved in https://github.com/BobbySoftwares/Bybbot/pull/7/commits/173d6bc07cf9a1a3121d5820f16d9bcb19a74a16